docs/src/InvariantTheory/finite_groups.md

```diff
@@ -91,6 +91,14 @@ julia> M2 = matrix(K, [1 0 0; 0 a 0; 0 0 -a-1])
 julia> G = matrix_group(M1, M2)
 Matrix group of degree 3
   over cyclotomic field of order 3
+with 2 generators
+  [0   0   1]
+  [1   0   0]
+  [0   1   0]
+
+  [1   0        0]
+  [0   a        0]
+  [0   0   -a - 1]
```

Comment:
Here it would be nice if there was a hint somewhere that `a` is the name of the generator of the cyclotomic field.

But I notice that the cyclotomic field has no option for printing that. Indeed, even its detailed printing omits the name of its generator:
```
julia> K, a = cyclotomic_field(3, "a")
(Cyclotomic field of order 3, a)

julia> K
Number field with defining polynomial _$^2 + _$ + 1
  over rational field

julia> gen(K)
a
```

In fact I am surprised to see that the detailed printing drops the hint that this is a cyclotomic field (I mean, of course one can see this from the defining polynomial, but in larger degrees I would not be able to recognize these reliably).